### ISSUE TITLE
Rédiger les spécification pour la page show Sector (thématique)

### ISSUE THREAD

**Issue by simonLouvet** [data-players/passerelle_normandie#93]

**Comment by BastienSig** [comment]
Pour la photo bannière, est ce que tu veux que ce soit la même que celle de la page d’accueil ou une nouvelle ? Un peu comme pour les organisations, logo + bannière ?

**Comment by ilouboff** [comment]
Effectivement, une première photo pour la bannière et une seconde photo pour la page d'accueil, ça semblerait plus pertinent. C'est top, merci!
